src/nile/artifacts/abis/Account.json

```
@@ -150,4 +150,4 @@
         ],
         "type": "function"
     }
-]
+]
```

```suggestion
]

```
Hmm I don't remember seeing the "no newline" error in the past here